js/bittrex.js

```diff
@@ -841,6 +867,21 @@ module.exports = class bittrex extends Exchange {
         return this.parseOrder (response['result']);
     }
 
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let request = {};
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = market['id'];
+        }
+        let response = await this.accountGetOrderhistory (this.extend (request, params));
```

Comment:
Fetching order history with fetchMyTrades may be wrong, we should do that in `fetchClosedOrders` instead, therefore we might have to rename this method from `fetchMyTrades` to `fetchClosedOrders`. See the difference here:

- https://github.com/ccxt/ccxt/wiki/Manual#how-orders-are-related-to-trades
- https://github.com/ccxt/ccxt/wiki/Manual#querying-orders